src/lj_ccall.h

```diff
@@ -86,10 +86,23 @@ typedef union FPRArg {
 #elif LJ_TARGET_PPC
 
 #define CCALL_NARG_GPR		8
+#if LJ_ARCH_BITS == 64
+#define CCALL_NARG_FPR		13
```

Comment:
Is this is a number of arguments to be passed in FPR, it should be 14, AFAIK, and not restricted to ppc64: 32-bit ppc also has 14 FPRs available:

> The registers that can be used to pass arguments to called functions are the general-purpose registers GPR3 through GPR10, the floating-point registers FPR1 through FPR13, and the vector registers V2 through V13 (see "Register Preservation" (page 23) for details).
http://personal.denison.edu/~bressoud/cs281-s07/MacOSXLowLevelABI.pdf